src/external_data_ifaces.cpp

```diff
@@ -32,4 +34,23 @@ void ExternalDataIFaces::bmcRedundancy(const BMCRedundancy& bmcRedundancy)
     _bmcRedundancy = bmcRedundancy;
 }
 
+SiblingBmcIP ExternalDataIFaces::siblingBmcIP() const
```

Comment:
This probably also could have been a const reference, but not that big of a deal.